stable/sonarqube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.10.3
+version: 0.10.4
```

Comment:
Please bump the minor version instead since this adds a feature

Reply:
done